### ISSUE TITLE
[Dev Version] start of the round desync

### ISSUE THREAD

**Issue by AbominationBanisher** [Regalis11/Barotrauma#1323]
desynced at the start of the round making me sit out the entire round, also applies to spectating. pretty darn peeved.

**Comment by AbominationBanisher** [comment]
made me miss yet another round, fuck it. it was on nilanth's crappy xml edited server too, for further detail

**Comment by NilanthAnimosus** [comment]
the xml edits are light (Job min counts, starter items, some tweaks to condition change rates in alien items, deconstruct items to spawn) and everybody else started the round fine except abomination and I think 1 other, so 5-6 successful clients and 1-2 not successful. these tweaks should sync with clients fine and not cause entity-related problems.

I'm not sure of the reason and I stated I would add some detail where I Could if possible, though and they could never rejoin. naturally pointing at the XML edits when really the real thing to state is I am using a number of code changes I sorted through myself to implement early.

such changes include a little custome code, and a number of commit changes like campaign ID syncing fixes, the ability to use flares works fine, not sending the entity changes for light components, I went through all the commits and cherrypicked by-hand information accurately I figured will sync with clients correctly and bugfix the server a version early.

my serverlogs indicate the following and shows evidence of it in the past for another player:

![image](https://user-images.githubusercontent.com/29177976/54723022-1736ff00-4b5e-11e9-88bd-49b2b83333ab.png)

this does not appear to be a problem of missing events and is isolated only to abomination, and complains of old, removed events.
